docs/tutorial/project_configuration.md

```diff
@@ -0,0 +1,56 @@
+
+1. For the current project, run the `ESP-IDF: Open ESP-IDF Terminal` command to launch an ESP-IDF loaded terminal.
+
+2. Run the `idf.py save-defconfig` command to generate a `sdkconfig.defaults` file.
+
+3. Use the `ESP-IDF: Open Project Configuration` and create a new profile with name `production`. Set `sdkconfig defaults` the previous `sdkconfig.defaults` file. If you want to separate generated binaries of this new **production** profile from the default case, specify a different build folder using the `Build Directory path` to something like `/path/to/current/project/build_production` and the `sdkconfig file path` to something like `/path/to/current/project/build_production/sdkconfig` to avoid mixing development sdkconfig with production sdkconfig files.
+
+4. Create a new profile with name `development`. You can set the build folder using the `Build Directory path` to something like `/path/to/current/project/build_dev` and the `sdkconfig file path` to something like `/path/to/current/project/build_dev/sdkconfig` to avoid mixing development sdkconfig with production sdkconfig files.
+
+5. After creating each profile and the configuration settings for each profile, click the `Save` button and use the `ESP-IDF: Select project configuration` command to choose the profile to override extension configuration settings.
+
+6. When you choose the **production** profile and use the `ESP-IDF: Build your project` the binaries are going to be created in `/path/to/current/project/build_production`. If you choose the **development** profile, the binaries are going to be created in `/path/to/current/project/build_dev`.
+
+7. These profiles and each profile settings are going to be saved in the `/path/to/current/project/esp_idf_project_configuration.json`.
+
+The production environment could be split into multiple production profiles, as it is done in the [ESP-IDF CMake Multiple configuration example](https://github.com/espressif/esp-idf/tree/master/examples/build_system/cmake/multi_config) and the [multiple configuration tutorial](./multiple_config.md) by separating common sdkconfig settings in a `sdkconfig.prod_common` and product specific settings in `sdkconfig.prod1` and `sdkconfig.prod2` respectively. Multiple sdkconfig defaults files can be specified in the project configuration editor profile `sdkconfig defaults` field as `sdkconfig.prod_common;sdkconfig.prod1` where the values are loaded in order as explained in [here](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/build-system.html?highlight=sdkconfig%20defaults#custom-sdkconfig-defaults).
```

Comment:
Ahh, now i get it.

before you were describing a simpler approach with fully separate sdkconfig input files. 

you need to move this section earlier!